src/microbe_stage/MicrobeHUD.cs

```diff
@@ -806,16 +818,16 @@ private void UpdateHealth(float delta)
             return;
 
         var hp = 0.0f;
-        var maxHP = 100.0f;
 
+        // Update to the player's current HP, unless the player is dead
         if (stage.Player != null)
```

Comment:
```suggestion
        if (stage.Player?.Dead == false)
```

Reply:
I committed this suggestion, then thought again and reverted it. The comment was the misleading part; it's really checking if the player is already disposed, since the original point of this PR is to still show the maxHP. I've kept the code the same and updated the comment.